src/keygen/KeygenOptions.hs

```diff
@@ -43,22 +50,22 @@ data FakeAvvmOptions = FakeAvvmOptions
     , faoOneStake    :: Word64
     } deriving (Show)
 
-optsParser :: Parser KeygenOptions
-optsParser = do
+optionsParser :: Parser KeygenOptions
+optionsParser = do
     koGenesisFile <- strOption $
         long    "genesis-file" <>
         metavar "FILE" <>
         value   "genesis.bin" <>
-        help    "File to dump binary shared genesis data"
+        help    "File to dump binary shared genesis data."
     koRearrangeMask <- optional $ strOption $
         long    "rearrange-mask" <>
         metavar "PATTERN" <>
-        help    "Secret keyfiles to rearrange"
+        help    "Secret keyfiles to re-arrange."
```

Comment:
It's "rearrange" after all:
https://answers.yahoo.com/question/index?qid=20121202112942AAzC1VU